module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -870,6 +864,63 @@ protected function getDueDate($itemId, $showTime)
         return '';
     }
 
+    /**
+     * Should we use the legacy authentication mechanism?
+     *
+     * @return bool
+     */
+    protected function useLegacyAuthentication(): bool
+    {
+        return $this->config['API']['legacy_authentication'] ?? false;
```

Comment:
This default `false` conflicts with the value in `Folio.ini`.  I would change it here as I think `useLegacyAuthentication` should be true in VF 10.0 -- and I don't think you need a 9.1.x for it.  My current understanding from Slack and watching a meeting recording, is that although the backend RTR code exists in Poppy it will be disabled by default, and I doubt many hosting providers will enable it since the UI side is not yet available.  And if VF 10 is released in April as planned, it will beat the Quesnalia release (planned for April 29th, but the past suggests it will slip at least a bit).  So RTR should work but we should default to use legacy for now.

Reply:
Thanks -- I've changed the default value as per your suggestion. I still think it's worth including this in the 9.1.1 release since that release is coming anyway, and this will give people a low-pain upgrade path if they need the functionality and aren't ready to go all the way to 10.0. But it's good to know that there's likely to be less rush/pressure than anticipated.

Reply:
Sure if 9.1.1 is happening anyway, I agree it's helpful for those who won't want to upgrade to 10 immediately.  (Immediately meaning before Ransoms is released in summer/fall).

Reply:
This also takes some pressure off if we find that we need extra time to get 10.0 completed due to the complexity of some of the work-in-progress.